lib/erlef_web/controllers/blog_controller.ex

```diff
@@ -7,12 +7,17 @@ defmodule ErlefWeb.BlogController do
   # not sure what we want to do with this yet.
   def index(conn, %{"topic" => topic}) do
     with {:ok, posts} <- list(topic) do
```

Comment:
I think we can do away with `with` here. I think we can just have a call to render with `posts: list(topic)`. In addition we could put a guard on this `when not is_nil(topic)` in the cases of trying to index a non-existent group we would end up defaulting to `"eef"`. Thoughts?

Reply:
So we have two index functions heads. If the topic is not defined in the url, the second index will call the first with a default topic "eef". On the other hand, if the topic does not exist or it is nil, the Blogs.Repo will return a empty list. The tuple comes from calling list(topic) which calls the Blogs.Repo like this {:ok, Erlef.Blogs.Repo.for_wg(topic)}.

I agree with you getting rid of the with clause and will drop the {:ok, posts} tuple.